syntax = "proto3";


message Task {
  enum Progress{
     NOT_STARTED = 0;
     IN_PROGRESS = 1;
     DONE = 2;
     TESTED = 3;
  }
  string project_name = 1;
  // name of task, used as identifier
  string name = 2;

  // A =description of the task
  string description = 3;

  // Unix UTC timestamp of approximately when this task was registered
  uint64 timestamp = 4;
}

message TaskContainer {
  repeated Task entries = 1;
}
